Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:testing
+RUN apt-get update && apt-get -y install git python-pip python-dev
```

Comment:
W dniu 02.05.2016 o 11:54, Tomasz Magulski pisze:
>> @@ -0,0 +1,8 @@
>> +FROM debian:testing
>> +RUN apt-get update && apt-get -y install git python-pip python-dev
> 
> ```
> RUN apt-get update
> RUN apt-get -y install git python-pip python-dev
> ```

Akurat złączenie tych dwóch komend ma sens - inaczej ryzykujesz to że
cache Dockera wejdzie Ci w drogę i między apt-get update a installem
minie kilka dni, bo np zmieniłeś linię z `apt-get -y install` ale tej z
apt-get update już nie.